CHANGELOG.unreleased.md

```diff
@@ -47,6 +47,7 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Host/Dashboard] Fix accent of tag list (PR [#8731](https://github.com/vatesfr/xen-orchestra/pull/8731))
```

Comment:
Until XO 6 is the default version, we keep the changelog in a sub-section to make it clear that it's not visible from XO 5:
```suggestion
- **XO 6:**
   - [Host/Dashboard] Fix color of tag list (PR [#8731](https://github.com/vatesfr/xen-orchestra/pull/8731))
```